yolov3_deepsort.py

```diff
@@ -1,23 +1,30 @@
 import os
+from os.path import splitext, basename, join, dirname, isdir, isfile
+
 import cv2
 import time
 import argparse
 import torch
 import numpy as np
+from tqdm import tqdm
+from warnings import warn
 
 from detector import build_detector
 from deep_sort import build_tracker
-from utils.draw import draw_boxes
+from utils.tools import tik_tok, is_video
+from utils.tools import compute_color_for_labels
 from utils.parser import get_config
+from utils.bbox_to_json import JsonBboxLogger
 
 
 class VideoTracker(object):
     def __init__(self, cfg, args):
         self.cfg = cfg
         self.args = args
+        self.json_logger = JsonBboxLogger()
         use_cuda = args.use_cuda and torch.cuda.is_available()
         if not use_cuda:
-            raise UserWarning("Running in cpu mode!")
+            warn(UserWarning("Running in cpu mode!"))

```

Comment:
Using raise function stops the program.
Maybe its best to warn the user without stopping program